cross/ffmpeg/Makefile

```diff
@@ -83,7 +122,18 @@ ifeq ($(findstring $(ARCH),$(PPC_ARCHES)),$(ARCH))
 DEPENDS += native/nasm
 ENV += PATH=$(NASM_PATH):$$PATH
 CONFIGURE_ARGS += --arch=ppc
+
 ifeq ($(findstring $(ARCH),qoriq),$(ARCH))
 CONFIGURE_ARGS += --disable-asm --cpu=e500v2
 endif
+
+# libshine - gcc to old on ppc853x: undefined reference to `__builtin_bswap16'
+ifneq ($(findstring $(ARCH),ppc853x),$(ARCH))
+DEPENDS += cross/shine
+CONFIGURE_ARGS += --enable-libshine
+endif
 endif
+
+.PHONY: synoheaderPostInstall
+synoheaderPostInstall:
+	@install -m 644 $(WORK_DIR)/$(PKG_DIR)/synoconfig.h $(WORK_DIR)/install$(INSTALL_PREFIX)/include
```

Comment:
I do not understand how an header file may be useful after compilation (post_install target). I simply wonder if this target is required.
If really required, please rename target as `ffmpeg_post_install`

Reply:
Done.
The post-install is only for evansport requirement as external library from toolkit requires it.
It isn't required for any other arches.